### PR TITLE
Changed cube transfer command

### PR DESCRIPTION
**Pull request by hetaskesen**
Used the cone commands to change the cube transfer command to use claw intake to better hold cube

### CODE REVIEW

File: src/main/java/competition/commandgroups/MoveCollectedGamepieceToArmCommandGroup.java [hunk 1]
```diff
@@ -58,10 +59,15 @@ public MoveCollectedGamepieceToArmCommandGroup(
                 () -> UnifiedArmSubsystem.KeyArmPosition.PrepareToAcquireFromCollector,
                 () -> UnifiedArmSubsystem.RobotFacing.Forward);
 
-        var waitThenPullArm = new WaitCommand(0.1).andThen(cubePullOutGamePieceCommand);
+        //Instead of waitThenPullArm, will use the clawConeIntake with a small change
+        //var waitThenPullArm = new WaitCommand(0.1).andThen(cubePullOutGamePieceCommand);
+        var clawCubeIntake = clawMotors.createIntakeCommand()
```
Comment (aschokking) [inline src/main/java/competition/commandgroups/MoveCollectedGamepieceToArmCommandGroup.java:64]: you create this command, but then never try to run it in the command group so it won't happen. Look at how the cone version is using it

File: src/main/java/competition/commandgroups/MoveCollectedGamepieceToArmCommandGroup.java [hunk 2]
```diff
@@ -58,10 +59,15 @@ public MoveCollectedGamepieceToArmCommandGroup(
...
+                .withTimeout(cubeClawIntakeTime)
+                .andThen(clawMotors.createStopCommand().withTimeout(0.01));
+
         var cubePullSequence = collector.getEjectThenStopCommand()
                 .withTimeout(cubeCollectorEjectTime)
-                .alongWith(waitThenPullArm);
+        //      .alongWith(waitThenPullArm); Would this be neccesary if the cube is secured in claw?
```
Comment (aschokking) [inline src/main/java/competition/commandgroups/MoveCollectedGamepieceToArmCommandGroup.java:70]: if not the collector might keep trying to pull the cube back?

**Review by aschokking — CHANGES_REQUESTED** [review]
Glad to have you back in the mix!

File: src/main/java/competition/commandgroups/MoveCollectedGamepieceToArmCommandGroup.java (hunk 2 above)
Comment (hetaskesen) [reply src/main/java/competition/commandgroups/MoveCollectedGamepieceToArmCommandGroup.java:70]: Ahh alright, adding that back

**Review by aschokking — APPROVED** [review]
This looks right to me 👍🏻

**Pull request merged**